modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -14579,6 +14579,10 @@ set-default-shipping-address=Set Default Shipping Address
 set-element-as-click-target=Set Element as Click Target
 set-expiration-date=Set Expiration Date
 set-filters=Set Filters
+set-lar-cleanup-job-interval=LAR Cleanup Job Execution Frequency
+set-lar-cleanup-job-interval-help=Configure execution time intervals for the LAR cleanp job in minutes. The default is every 15 minutes.
+set-lar-expiry-in-days=LAR Expiry
```

Comment:
May reword this key to `export-import-entry-expiry-days`, and the text could be also `Export/Import Entry expiry days`